nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -248,8 +235,6 @@ class ConfigEncryptionTool {
         options.addOption(Option.builder("i").longOpt(OUTPUT_LOGIN_IDENTITY_PROVIDERS_ARG).hasArg(true).argName("file").desc("The destination login-identity-providers.xml file containing protected config values (will not modify input login-identity-providers.xml)").build())
         options.addOption(Option.builder("a").longOpt(AUTHORIZERS_ARG).hasArg(true).argName("file").desc("The authorizers.xml file containing unprotected config values (will be overwritten unless -u is specified)").build())
         options.addOption(Option.builder("u").longOpt(OUTPUT_AUTHORIZERS_ARG).hasArg(true).argName("file").desc("The destination authorizers.xml file containing protected config values (will not modify input authorizers.xml)").build())
-        options.addOption(Option.builder("f").longOpt(FLOW_XML_ARG).hasArg(true).argName("file").desc("The flow.xml.gz file currently protected with old password (will be overwritten unless -g is specified)").build())
-        options.addOption(Option.builder("g").longOpt(OUTPUT_FLOW_XML_ARG).hasArg(true).argName("file").desc("The destination flow.xml.gz file containing protected config values (will not modify input flow.xml.gz)").build())
```

Comment:
These options need to be retained and changed to indicate flow.json.gz. We will need separate work on the toolkit in general, but at least for this PR, it should continue to function with flow.json.gz.

Reply:
Yes, as mentioned in the description, originally I wanted to do the toolkit changes in a follow up story after clarifying the cleanup and general changes within this PR. Depending on your answer for my comment above hovere I will change this part accordingly. For further toolkit related changes I suggest to move them in a followup PR.

Reply:
Thanks for the reply @simonbence. Handling the toolkit changes in a separate PR sounds good, but in that case, I recommend reverting this particular removal and keeping the toolkit changes as minimal as possible. As these argument names do not impact actual flow.xml.gz usage, leaving them in place seems fine for now, and more substantive changes can be handled separately. So from that perspective, I recommend limiting the toolkit changes to only those that are required for the removal of actual flow.xml.gz handling code.